community/bitcoin/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=bitcoin
-pkgver=0.15.0.1
+pkgver=0.15.1
 _ver=${pkgver/_/}
 pkgrel=3
```

Comment:
`pkgrel` should be reset to `0` on pkgver bumps.

Reply:
sorry - fixed & rebased